pkg/katatestutils/README.md

```diff
@@ -0,0 +1,116 @@
+# Kata test utilities
+
+* [Test Constraints](#test-constraints)
+    * [Usage](#usage)
+    * [Examples](#examples)
+        * [Skip tests based on user](#skip-tests-based-on-user)
+        * [Skip tests based on distro](#skip-tests-based-on-distro)
+    * [Full details](#full-details)
+
+This package provides a small set of test utilities.
+
+## Test Constraints
+
+This package provides helper functions to allow tests to be skipped based on a
```

Comment:
Lines 14-15 suggested rewrite (unless I've changed the meaning - previous sentence is a bit ambiguous): 
This package provides helper functions that accept user-specified constraints that allow you to skip tests.